src/store/modules/GlobalStore.js

```diff
@@ -1,10 +1,35 @@
 import api from '../api';
 
+const HOST_STATE = {
+  on: 'xyz.openbmc_project.State.Host.HostState.Running',
+  off: 'xyz.openbmc_project.State.Host.HostState.Off',
+  error: 'xyz.openbmc_project.State.Host.HostState.Quiesced',
+  transition: {
+    on: 'xyz.openbmc_project.State.Host.Transition.On',
+    reboot: 'xyz.openbmc_project.State.Host.Transition.Reboot'
+  }
+};
+
+const hostStateMapper = hostState => {
+  switch (hostState) {
+    case HOST_STATE.on:
+      return 'Running';
+    case HOST_STATE.off:
+      return 'Off';
+    case HOST_STATE.error:
+      return 'Error';
+    case HOST_STATE.transition.reboot:
```

Comment:
how would we ever hit this case? 
HostState can only ever be running, off, or Quiesced, so we hit those first...

https://github.com/openbmc/phosphor-dbus-interfaces/blob/master/xyz/openbmc_project/State/Host.interface.yaml#L35

Reply:
I added these transition states because they were being returned with the websocket events. I've seen transition.on and transition.reboot

Reply:
I'm checking for either CurrentHostState or RequestedHostTransition in the websocket event